src/ast/irbuilderbpf.h

```diff
@@ -171,6 +171,7 @@ class IRBuilderBPF : public IRBuilder<>
                        AllocaInst *data,
                        Value *data_len,
                        const location &loc);
+  void hoist(const std::function<void()> &functor);
```

Comment:
Can you leave a comment describing the `hoist()` semantics? Maybe "hoist() hoists instructions generated by `functor` to the start of the containing function".

Reply:
I've now added a comment here.